service/src/main/resources/templates/ehr_request_statement_template.mustache

```diff
@@ -7,5 +7,8 @@
         {{{availabilityTime}}}
         {{{defaultReasonCode}}}
         <text>{{description}}</text>
+        {{#code}}
```

Comment:
Do we need the code section here?

Reply:
ReasonCode has 0..* cardinality and if there isn't a block here it leaves a blank line that breaks the tests